apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/layout.tsx

```diff
@@ -60,7 +70,15 @@ export default async function ConnectedEvaluationLayout({
             documentUuid={params.documentUuid}
           />
         }
-        table={<></>}
+      />
+
+      <MetricsSummary
```

Comment:
Why not inside `table` from TableWithHeader?

Reply:
idk, feels misleading using the `table` attribute for non-table components...